code/modules/admin/verbs/infiltratorteam_syndicate.dm

```diff
@@ -0,0 +1,199 @@
+// Syndicate Infiltration Team (SIT)
+// A little like Syndicate Strike Team (SST) but geared towards stealthy team missions rather than murderbone.
+
+var/global/sent_syndicate_infiltration_team = 0
+
+/client/proc/syndicate_infiltration_team()
+	set category = "Event"
+	set name = "Send Syndicate Infiltration Team"
+	set desc = "Spawns a squad of syndicate infiltrators on the Syndicate Mothership if you want to run an admin event."
+	if(!check_rights(R_ADMIN))
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(!ticker)
+		alert("The game hasn't started yet!")
+		return
+	if(alert("Do you want to send in the Syndicate Infiltration Team?",,"Yes","No")=="No")
+		return
+	var/spawn_dummies = 0
+	if(alert("Spawn full-size team, even if there aren't enough ghosts to populate them?",,"Yes","No")=="Yes")
+		spawn_dummies = 1
+	var/list/teamsizeoptions = list(1,2,3,4,5)
+	var/teamsize = input(src, "How many team members, not counting the team leader?") as null|anything in teamsizeoptions
+	if (!(teamsize in teamsizeoptions))
+		alert("Invalid team size specified. Aborting.")
+		return
+	var/input = null
+	while(!input)
+		input = sanitize(copytext(input(src, "Please specify which mission the syndicate infiltration team shall undertake.", "Specify Mission", ""),1,MAX_MESSAGE_LEN))
+		if(!input)
+			alert("No mission specified. Aborting.")
+			return
+	var/list/tcoptions = list(0,5,10,15,20,25,30,35,40,50,60,80,100)
+	var/tcamount = input(src, "How much TC do you want to give each team member? Suggested: 20-30. They cannot trade TC.") as null|anything in tcoptions
+	if (!(tcamount in tcoptions))
+		alert("Invalid TC amount specified. Aborting.")
+		return
+	var/spawn_sit_officer = 0
+	if(alert("Spawn a syndicate officer for you, so you can brief them before they go?",,"Yes","No")=="Yes")
+		spawn_sit_officer = 1
+	if(sent_syndicate_infiltration_team == 1)
+		if(alert("A Syndicate Infiltration Team has already been sent. Sure you want to send another?",,"Yes","No")=="No")
+			return
+
+	var/syndicate_leader_selected = 0
+	//var/list/infiltrators = pollCandidates("Do you want to play as a SYNDICATE INFILTRATOR?", ROLE_TRAITOR, 7)
+	to_chat(src, "Polling candidates...")
+	var/list/infiltrators = pollCandidates("Do you want to play as a SYNDICATE INFILTRATOR?")
```

Comment:
You might just want to make this so admins can pick the members of the team, similar to death squads and honk squads.